### PR TITLE
[Message] centered and right aligned variant

### PR DESCRIPTION
**Pull request by lubber-de**
## Description
This PR adds a `centered` or `center aligned` as well as a `right aligned` variant to the `message` component just like many other components already support.

## Testcase
https://jsfiddle.net/lubber/03qx4gr5/39/

## Screenshot
image

## Closes
https://github.com/Semantic-Org/Semantic-UI/issues/5049

### CODE REVIEW

**Review by exoego — APPROVED** [review]
LGTM

**Review by prudho — COMMENTED** [review]
Nice ! But the icon message seems a little away now.

Capture

Maybe we should align it with content ?

**Comment by lubber-de** [comment]
> Maybe we should align it with content ?

@prudho Fixed by edbc0c4586b01795ed12dcce4d57457d5b8ba2ad 😉 
Updated jsfiddle and screenshot in issue description accordingly. Please review again

File: src/definitions/collections/message.less [hunk 1]
```diff
@@ -173,6 +173,10 @@
   .ui.centered.message,
   .ui.center.aligned.message {
     text-align: center;
+    justify-content: center;
+    & > i.icon + .content {
+      flex: 0 0 auto;
+    }
```
Comment (ko2in) [inline src/definitions/collections/message.less]: I think it's better to omit adjacent sibling selector for `content`. So, we can also place icon at the right side (https://jsfiddle.net/ko2in/8r9axfjo/2/). The icon left margin needs to be defined in this case. Maybe, another PR for the icon's margin adjustments.

Comment (lubber-de) [reply src/definitions/collections/message.less]: @ko2in Thanks for the hint! I adjusted it according to your suggestion by 0bdd81fe4614ec38321ed69b2240b3677ac8a261
Please review again 🙂

Comment (lubber-de) [reply src/definitions/collections/message.less]: And while the right icon also works now, i added a `right aligned` variant as well to complete it 🙂

**Review by prudho — APPROVED** [review]
Well, you're really on fire today 😄 

LGTM !

**Review by ko2in — APPROVED** [review]
LGTM :+1: My mailbox is flooding today about your PR notifications :smile:

**Comment by lubber-de** [comment]
Docs added by https://github.com/fomantic/Fomantic-UI-Docs/pull/252

**Pull request merged**